### PR TITLE
Post Ratings + tests + view ratings

### PR DESCRIPTION
**Pull request by pmasilva20**

### CODE REVIEW

File: mainApp/src/main/java/logisticsmarshall/tqs/ua/controllers/LogisticsAPIController.java [hunk 1]
```diff
@@ -158,6 +160,28 @@
...
+        Delivery delRequested = deliveryService.getDeliveryById(delivery_id);
+        Driver driverAssigned = driverService.getDriverById(driver_id);
+
+        Reputation rep = new Reputation();
+        rep.setDelivery(delRequested);
+        rep.setRating(delivery_id);
+        rep.setDriver(driverAssigned);
+        rep.setDescription(description);
+        rep.setRating(rating);
+
```
Comment (buckaroo69) [inline mainApp/src/main/java/logisticsmarshall/tqs/ua/controllers/LogisticsAPIController.java:180]: you probably need to set delivery.reputation and save it too or it'll fail when overwriting a review

**Review by buckaroo69 — CHANGES_REQUESTED** [review]
see thing I commented

File: mainApp/src/main/java/logisticsmarshall/tqs/ua/controllers/LogisticsAPIController.java (hunk 1 above)
Comment (pmasilva20) [reply mainApp/src/main/java/logisticsmarshall/tqs/ua/controllers/LogisticsAPIController.java:180]: sure

**Review by buckaroo69 — APPROVED** [review]
ok very cool

**Pull request merged**